test/generateprecompiles.jl

```diff
@@ -0,0 +1,26 @@
+# To figure out what should be precompiled, run this script, then move
+# /tmp/precompile/precompile_Plots.jl to src/precompile.jl
+
+using SnoopCompile
+
+# run examples with GR backend, logging what needs to be compiled
+SnoopCompile.@snoopc "/tmp/compiles.log" begin
+    using Plots
+    Plots.test_examples(:gr, disp=true)
```

Comment:
You could add `skip=[25, 30]` in `test_examples` to avoid the `@df` warnings.

Reply:
I've just added `using StatsPlots` to those examples, so they work now.